app.py

```diff
@@ -13,9 +14,15 @@
                   static_folder='dist/client', static_url_path='/client/')
 app.register_blueprint(example.blueprint)
 
+# Initialize Firestore DB
+cred = credentials.Certificate('key.json')
+default_app = initialize_app(cred)
+db = firestore.client()
+todo_ref = db.collection('characters')
```

Comment:
there is likely  a better name for this, character_query, etc

Reply:
Remaned todo_ref to character_data.